Docs/sphinx_documentation/source/Testing.rst

```diff
@@ -18,6 +18,7 @@ application codes that use it as a framework. We use an in-house test runner scr
 operation, originally developed by Michael Zingale for the Castro code, and later expanded to other
 application codes as well. The results for each night are collected and stored on a web page; see
 https://ccse.lbl.gov/pub/RegressionTesting/ for the latest set of results.
+The runtime option ``abort_on_unused_inputs`` (``0`` or ``1``; default is ``0`` for false) is useful for making sure that tests always stay up to date with API changes as it will abort the application after the test run if any unused input parameters were detected.
```

Comment:
Ist it the param parse option prefixed `amrex...`?

Reply:
Yes, @ax3l is correct.  It should be `amrex.abort_on_unused_inputs`.  @n01r Could you make the change?

Reply:
Yep, I will, thanks. I somehow thought it only had the prefix in the applications using AMReX but I see now that there is a `ParmParse pp("amrex");` also in `AMReX.cpp`.